README.md

```diff
@@ -1,33 +1,25 @@
-<!--
-parent:
-  order: false
--->
+# exocore
 
-Exocore is a scalable, high-throughput Proof-of-Stake blockchain
-that is fully compatible and interoperable with Ethereum.
-It's built using the [Cosmos SDK](https://github.com/cosmos/cosmos-sdk/)
-which runs on top of the [Tendermint Core](https://github.com/tendermint/tendermint) consensus engine.
-
-## Quick Start
-
-To learn how Exocore works from a high-level perspective.
-
-TBD
+Exocore is an omnichain restaking protocol that combines the crypto-economic security from multiple blockchain networks and tokens, and extends it to any off-chain system. The protocol is designed with a modular architecture, incorporating Tendermint-based Byzantine Fault Tolerant (BFT) consensus mechanism, Zero-Knowledge (ZK) light-client bridging, and a fully EVM-compatible execution environment. This design enables smooth interactions for restakers and seamless integration for developers. Additionally, we introduce novel concepts, such as Union Restaking, where off-chain services can form a union to extend the crypto-economic security of their own tokens to each other. By pooling crypto-economic security and extending it to off-chain systems, Exocore powers an open market for decentralized trust.
 
 ## Documentation
-TBD
-
-## Installation
-Once the dependencies are installed, run:
-
-```bash
-make install
-```
-
-Or check out the latest [release](https://github.com/Exocore/Exocore/releases).
-
-## Community
-TBD
 
-## Contributing
-TBD
\ No newline at end of file
+To learn how Exocore works from a high-level perspective, see the [Exocore Whitepaper](https://t.co/A4y4YcOuEC)
```

Comment:
Please change the URL to the direct link https://docs.exocore.network/whitepaper/.pdf